pantheon/src/test/java/tech/pegasys/pantheon/cli/subcommands/blocks/BlockSubCommandTest.java

```diff
@@ -64,11 +69,13 @@
           + System.lineSeparator();
 
   private static final String EXPECTED_BLOCK_IMPORT_USAGE =
-      "Usage: pantheon blocks import [-hV] --from=<FILE>"
+      "Usage: pantheon blocks import [-hV] [--format=<format>] --from=<FILE>"
           + System.lineSeparator()
           + "This command imports blocks from a file into the database."
           + System.lineSeparator()
-          + "      --from=<FILE>   File containing blocks to import"
+          + "      --format=<format>   The type of data to be imported (default: RLP)."
```

Comment:
Is there somewhere the user can see that JSON is a valid option to this flag?

Reply:
Good call - will update!